src/Charts/Pie/pie.scss

```diff
@@ -0,0 +1,105 @@
+@import '../../Utilities';
+@import '../../Utilities/variables';
```

Comment:
You shouldn't need to import `../../Utilities/variables` because `Utilities.scss` imports that for you.

Reply:
fixed in both Donut and Pie